packages/claycss.com/electric.config.js

```diff
@@ -32,10 +32,10 @@ module.exports = {
 		return generateIconData(data);
 	},
 	codeMirrorLanguages: ['xml', 'htmlmixed', 'soy'],
-	metalComponents: ['electric-quartz-components'],
+	metalComponents: ['electric-quartz-components', 'clay-charts'],
```

Comment:
While we're at it, we might as well add the rest of clay components, ideally in a programatic way 😉

Reply:
That sounds like a better idea, thanks.